### ISSUE TITLE
orig_id is NULL for some samples & acquisitions

### ISSUE THREAD

**Issue by grololo06** [oceanomics/ecotaxa_dev#540]
This comes from a mandatory column as per specs so it should not be.

```
ecotaxa=# select count(*) from samples where orig_id is null;
 count 
-------
   478
(1 ligne)

ecotaxa=# select count(*) from acquisitions where orig_id is null;
 count 
-------
 48535
(1 ligne)

```

**Comment by grololo06** [comment]
To link with #546 during solve

**Comment by grololo06** [comment]
`select distinct projid from samples where orig_id is null`
-> 3381 only
`select distinct projid from acquisitions where orig_id is null`
-> 939 only

**Comment by grololo06** [comment]
Solved in 2.5.0
